### PR TITLE
History

### PR DESCRIPTION
**Pull request by gberaudo**

### CODE REVIEW

**Comment by gberaudo** [comment]
See: https://github.com/c2corg/v6_ui/issues/48

**Comment by gberaudo** [comment]
Ready for review.

File: c2corg_ui/views/route.py [hunk 1]
```diff
@@ -46,3 +46,8 @@ def edit(self):
             'route_id': id
         })
         return self.template_input
+
+    @view_config(route_name=_API_ROUTE + '_history',
```
Comment (asaunier) [inline c2corg_ui/views/route.py]: Why using _API_ROUTE in the route_name whereas it has not been used for other route_names?
I think it is unnecessarly confusing, I would prefer we use the explicit route_name here.

File: c2corg_ui/views/waypoint.py [hunk 2]
```diff
@@ -47,3 +47,8 @@ def edit(self):
             'waypoint_id': id
         })
         return self.template_input
+
+    @view_config(route_name=_API_ROUTE + '_history',
```
Comment (asaunier) [inline c2corg_ui/views/waypoint.py]: Ditto

File: c2corg_ui/views/document.py [hunk 3]
```diff
@@ -115,6 +115,26 @@ def _get_filter_params(self):
                     params.append(filters[i:i+2])
         return params
 
+    def _get_history(self):
+        id, culture = self._validate_id_culture()
+        url = '%s/%s/%d/%s/%s' % (
+            self.settings['api_url'], 'document', id, 'history', culture)
```
Comment (asaunier) [inline c2corg_ui/views/document.py]: Why not simply using
```
url = '%s/document/%d/history/%s' % (self.settings['api_url'], id, culture)
```
?

File: c2corg_ui/templates/document/history.html [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+      <td>Radios ${v['version_id']}</td>
+      <td><a href="${request.route_url(module+'_view', id=document_id, culture=culture)}#TODOversion">{{'${1000 * v['written_at']}' | date:'medium'}}</a></td>
+      <td><a href="#TODO profile for ${v['user_id']}">${v['username']}</a></td>
+      <td>${v['comment']}</td>
+    </tr>
+  % endfor
+</table>
+
+<ul>
+  <li><a href="${request.route_url(module+'_view', id=document_id, culture=culture)}" translate>View</a></li>
```
Comment (asaunier) [inline c2corg_ui/templates/document/history.html]: Wouldn't flake8 require blank space around "+"?

**Comment by gberaudo** [comment]
PR updated.

**Comment by asaunier** [comment]
You also have a conflict with the PO files I guess. I'm trying to fix it in my PR too: #51 
I don't know if we should try to fix it in parallel or if it is better to wait that it's OK for one of us.

**Comment by gberaudo** [comment]
OK, waiting for your merge.

**Comment by asaunier** [comment]
My PR is merged.

**Comment by gberaudo** [comment]
Rebased, thanks.

**Pull request merged**